source/features/hide-markdown-diff.tsx

```diff
@@ -20,6 +35,7 @@ async function init(): Promise<void> {
 		</div>
 	);
 	delegate(document, '.rgh-preview-button:not(.selected)', 'click', togglePreviewResult);
+	new MutationObserver(fixDuplicatedContent).observe((await elementReady('.js-code-editor'))!, {attributeFilter: ['class']});
```

Comment:
Couldn't find an event corresponding to the preview finishing its loading, so I used a `MutationObserver`.